generator/tree_test.go

```diff
@@ -284,27 +284,27 @@ func TestGenerateConfigModule(t *testing.T) {
 				Walk: []string{"1"},
 				Metrics: []*config.Metric{
 					{
-						Name: "tableCreateDescr",
+						Name: "tableCreate",
 						Oid:  "1.1.1.2",
-						Type: "OctetString",
+						Type: "gauge",
 					},
 					{
-						Name: "tableWriteOnlyId",
+						Name: "tableWriteOnly",
```

Comment:
I might get the answer. Due to our introduced `access` mechanism, our test cases also generate its own access code, and things doesn't work anymore...